src/components/NavFooter.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div class="container">
+    <div class="row">
+      <div class="footer-col">
+        <ul>
+          <li>
+            <img src="../assets/images/logo.svg" alt="logo" />
+          </li>
+        </ul>
+      </div>
+      <div class="footer-col">
+        <h4>Products</h4>
+        <ul>
+          <li><a href="#">Quiz-App</a></li>
+        </ul>
+      </div>
+      <div class="footer-col">
+        <h4>Created by</h4>
+        <ul>
+          <li><a href="https://github.com/helenahalldiniths">Helena</a></li>
+          <li><a href="https://github.com/Philippevial">Philippe</a></li>
+          <li><a href="https://github.com/FelixJacobsen">Felix</a></li>
+          <li><a href="https://github.com/ErikssonF">Fredrik</a></li>
+          <li><a href="https://github.com/LordRekishi">Patrik</a></li>
+        </ul>
+      </div>
+      <div class="footer-col">
+        <h4>Help and support</h4>
+        <ul>
+          <li><a href="mailto:felix.jacobsen@iths.se">Email Us</a></li>
+          <li><a href="#">Common questions</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="copyright-label">
+      <label>© Copyright 2022 Paefax. All rights reserved.</label>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+/* Import our font instead */
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
```

Comment:
I suggest putting this in the top of main.css instead since we will (probably) use this for more of the page later